print(floor(1.2));

try {
	floor("abc");
} catch Int => error {
	print(error);
}